mayastor/src/bdev/util/uri.rs

```diff
@@ -54,3 +60,32 @@ pub(crate) fn boolean(
     }
     value.parse::<bool>()
 }
+
+pub(crate) fn uuid(
+    value: Option<String>,
+    url: &Url,
+) -> Result<Option<uuid::Uuid>, NexusBdevError> {
+    Ok(match value {
+        Some(value) => Some(uuid::Uuid::parse_str(&value).context(
+            nexus_uri::UuidParamParseError {
+                uri: url.to_string(),
+                parameter: "uuid".to_string(),
+            },
+        )?),
+        None => None,
+    })
+}
+
+pub(crate) fn dev_set_uuid(name: &str, uuid: Option<uuid::Uuid>) {
+    match Bdev::lookup_by_name(&name) {
+        Some(mut bdev) => {
+            bdev.set_uuid(uuid.map(|u| u.to_string()));
+        }
+        // Should we fail to create a device if we can't set
```

Comment:
As mentioned I don't think this layer adds much value to working with bdevs however, if you add more "utils" then we can reassess when you're further down the road with it. However, we can not fail setting the UUID as do a simply memcopy